project/templates/articles/management.html

```diff
@@ -26,29 +26,33 @@
         });
       });
     </script>
-    <div>
-        <a href="{{url_for('article.create')}}">create</a>
-    </div>
-    <!-- Article table -->
-    <table>
-        <tr>
-            <th>article_id</th>
-            <th>enabled?</th>
-            <th>actions</th>
-        </tr>
-        {% for article in articles %}
+
+    <section>
+        <div>
+            <a href="{{url_for('article.create')}}">➕ create</a>
```

Comment:
That yellow + with yellow background has no contrast. What do you think? Have you tried with other icon colours?

Reply:
What? 
![image](https://user-images.githubusercontent.com/45942798/81471472-a6372880-91f1-11ea-98a3-720fa0d62571.png)

Reply:
As we have talked this morning we should change most of icons for an static asset, such as font-awesome.

Reply:
New improvement task open to do this.